src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -20,6 +20,7 @@ import * as ReportActionContextMenu from './ContextMenu/ReportActionContextMenu'
 import VirtualKeyboard from '../../../libs/VirtualKeyboard';
 import * as EmojiUtils from '../../../libs/EmojiUtils';
 import reportPropTypes from '../../reportPropTypes';
+import addEmojiToComposerTextInput from '../../../libs/addEmojiToComposerTextInput';
```

Comment:
Side note (future refactor): 

It seems this ` ReportActionItemMessageEdit.js` is just a stripped down version of `ReportActionCompose.js`
Logic is duplicated, meaning we always have to copy/paste fixes here 
It should be possible to capture the core logic here and build on top of that in the main composer component

Reply:
I would leave that to a future PR?